src/conversions.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FEATUREDETECT_SRC_CONVERSIONS_H_
+#define FEATUREDETECT_SRC_CONVERSIONS_H_
+
+#include <opencv2/opencv.hpp>
```

Comment:
I think, you need `opencv2/core.hpp` here

Reply:
it still building without, but i ll do that

Reply:
You can remove `#include <opencv2/opencv.hpp>` here. I meant that you should have been replaced one with another.

Reply:
oh i am so sorry

Reply:
done